src/main/java/com/client/core/base/workflow/node/AbstractWorkflowAction.java

```diff
@@ -41,39 +46,7 @@ protected String getMessageUsingKey(String key) throws NoSuchMessageException {
     }
 
     protected String getMessageUsingKey(String key, Object[] args) throws NoSuchMessageException {
-        return getMessageSource().getMessage(key, args, Locale.US);
-    }
-
-    protected MessageSource getMessageSource() {
-        if(this.messageSource == null) {
-            this.messageSource = AppContext.getApplicationContext().getBean("messageSource", MessageSource.class);
-        }
-
-        return messageSource;
-    }
-
-    protected EmailTemplateService getEmailTemplateService() {
-        if(this.emailTemplateService == null) {
-            this.emailTemplateService = AppContext.getApplicationContext().getBean(EmailTemplateService.class);
-        }
-
-        return this.emailTemplateService;
-    }
-
-    protected ApplicationSettings getAppSettings() {
-        if(this.appSettings == null) {
-            this.appSettings = AppContext.getApplicationContext().getBean(ApplicationSettings.class);
-        }
-
-        return this.appSettings;
-    }
-
-    protected Emailer getEmailer() {
-        if(this.emailer == null) {
-            this.emailer = AppContext.getApplicationContext().getBean(Emailer.class);
-        }
-
-        return emailer;
```

Comment:
These getters need to comeback - we have no knowledge of how many projects depend on them

Reply:
I knew I left a comment on this file for a reason